packages/sdk-components-react/src/__generated__/vimeo.props.ts

```diff
@@ -2,10 +2,8 @@ import type { PropMeta } from "@webstudio-is/generate-arg-types";
 
 export const props: Record<string, PropMeta> = {
   slot: { required: false, control: "text", type: "string" },
-  style: { required: false, control: "text", type: "string" },
   title: { required: false, control: "text", type: "string" },
   defaultChecked: { required: false, control: "boolean", type: "boolean" },
-  defaultValue: { required: false, control: "text", type: "string" },
```

Comment:
defaultValue shouldn't be removed?

Reply:
Seems like should be defined by component as html one is very generic.

Reply:
Yeah, I wonder if our inference is inaccurate or the types are actually inaccurate from the source

Reply:
Not sure why defaultValue is defined on all components even div
```
        defaultValue?: string | number | ReadonlyArray<string> | undefined;
```